{"uuid":"ee87d5d1-3bb1-482a-8a7f-d7cca6db1f56","historyId":"54167b5c99fc86eb3d459224e4369d3e","fullName":"tests.LoginTests.tc01_standardUser","labels":[{"name":"package","value":"tests.LoginTests"},{"name":"testClass","value":"tests.LoginTests"},{"name":"testMethod","value":"tc01_standardUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AutomationProject"},{"name":"subSuite","value":"tests.LoginTests"},{"name":"host","value":"LAPTOP-RMK290R9"},{"name":"thread","value":"32344@LAPTOP-RMK290R9.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"tc01_standardUser","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3","trace":"java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:153)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:767)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:704)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:70)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:49)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:904)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\r\n\tat org.testng.TestRunner.run(TestRunner.java:603)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\r\n\tat org.testng.TestNG.run(TestNG.java:1060)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3\r\n\tat tests.LoginTests.getDataFromExcel(LoginTests.java:36)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1685831017343,"stop":1685831017345}